src/Services/OrderService/OrderService.Application/OrderList/GetList/GetListByIdQuery.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Bookmarks.Domain.Wishlists;

```

Comment:
See comment from before. Ideally, every single service should not know of the existence of other services, and definitely not depend on them.